web_widget_float_highlight/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 ONESTEiN BV (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Highlight Float Field',
+    'images': [],
+    'category': 'Extra Tools',
+    'version': '10.0.1.0.0',
```

Comment:
Sorry but I don't agree with this definition of version... too early to write "10".